bnr-utils/nz_azConnector/nz_azConnector.go

```diff
@@ -404,6 +406,7 @@ func main() {
             close(work)
             <- done
             handleErrors(err)
+            fmt.Println("Upload successful. Total files uploaded:", filesuploaded)
             log.Println("Upload successful. Total files uploaded:", filesuploaded)
         }

```

Comment:
general, we should set logger as Aniket said to write at both the places.

Reply:
Done